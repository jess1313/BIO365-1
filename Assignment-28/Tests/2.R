suppressPackageStartupMessages(suppressWarnings(suppressMessages(source("2.R"))))

iris2 = iris
iris2$Species <- as.character(iris2$Species)
iris2 <- dplyr::filter(iris2, Species %in% c("virginica", "versicolor"))

print(classify(iris2, "Species", "classif.svm", list(mlr::acc, auc)))