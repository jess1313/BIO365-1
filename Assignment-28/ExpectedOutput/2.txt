      acc       auc 
0.9400000 0.7958333 
